LEGO1/pizza.cpp

```diff
@@ -1,13 +1,35 @@
 #include "pizza.h"
 
-// OFFSET: LEGO1 0x10037ef0 STUB
+// OFFSET: LEGO1 0x10037ef0
 Pizza::Pizza()
 {
-  // TODO
+  // FIXME: This inherits from LegoActor, probably why this isn't matching
+  this->m_unk80 = 0;
+  this->m_unk84 = 0;
+  this->m_unk88 = 0;
+  this->m_unk8c = -1;
+  this->m_unk98 = 0;
+  this->m_unk90 = 0x80000000;
 }
 
-// OFFSET: LEGO1 0x10038100 STUB
+// OFFSET: LEGO1 0x1002c7cf STUB
+// MxS32 IsleActor::Notify(MxParam *)
+// {
+//   // TODO
+//   return 0;
+// }
+
+// OFFSET: LEGO1 0x10038100
 Pizza::~Pizza()
 {
-  // TODO
+  // FIXME: some vtable call from tickle manager, unimplemented atm
+  delete this;
```

Comment:
You don't need to write `delete this`, that's the compiler's responsibility to handle destruction of the object after calling the destructor.

Reply:
thanks!! Once that's done I'll update this PR